README.md

```diff
@@ -1,2 +1,60 @@
-# OpenDream RFC Repository
-WIP
+# OpenDream RFC Repo
+
+This repository stores RFCs for the [OpenDream](https://github.com/OpenDreamProject/OpenDream) project. Current and future users of OpenDream are strongly encouraged to watch this repository and provide constructive input on all open pull requests.
+
+## Motivation
+
+Although the core goal of OpenDream is to provide one-way parity with BYOND, there are [many opportunities](https://github.com/OpenDreamProject/OpenDream/wiki/Differences-Between-OpenDream-and-BYOND#enhancements) in which OpenDream and the DM language can be improved while still enabling users to migrate from BYOND with minimal effort. Or to put it another way, we can add new functionality *on top* of BYOND parity so that OpenDream users can take advantage of them while BYOND users are not impacted. 
+
+A key example of this is [pragmas](https://github.com/OpenDreamProject/OpenDream/wiki/Pragmas---Error-Emissions). Pragmas provide configurable compiler emissions for things such as runtime errors that *can* be detected at compiletime but BYOND fails to detect. Utilizing pragmas while maintaining BYOND compatibility is as simple as declaring them in their own code file and putting this in your code somewhere:
+```
+#ifdef OPENDREAM
+// Don't include the file of pragmas unless `OPENDREAM` is defined (which OpenDream defines for you)
+#include("/path/to/pragma/file.dm")
+#endif
+```
+
+Furthermore, OpenDream is intended to provide a solution for *everyone* and not any specific codebase. To prevent any one person from turning the project into their own new programming language, this RFC process exists to solicit input from the wider community on any major changes that would impact their current or future usage of OpenDream.
```

Comment:
Should "major changes" be defined (e.g Breaking parity with byond) or is it a judgement thing?

I see the section below but that looks like it's supposed to be a "Questions to ask during the RFC" rather than a "when is an RFC necessary".

I've seen it several times before that someone puts a massive amount of effort into something they personally believe is a good change, and after it's PRed they find out it's not something that's wanted. Seems like something we should be addressing sooner rather than later

Reply:
It's a judgement thing. People choosing to invest a lot of effort into something without asking if it's wanted is their problem; it never hurts to ask.